lib/segment/src/vector_storage/tests/async_raw_scorer.rs

```diff
@@ -79,6 +79,8 @@ fn test_async_raw_scorer(
         test_random_score(&mut rng, &storage, id_tracker.deleted_point_bitslice())?;
     }
 
+    set_async_scorer(false);
```

Comment:
I think this is data racing with concurrent tests.

I'll be able to fix this tomorrow.

Reply:
Resolved by 8a8469d06d2cab0f0af6bff602414e0847d381f5